node/cn/gasprice/gasprice.go

```diff
@@ -115,7 +115,7 @@ func NewOracle(backend OracleBackend, config Config, txPool TxPool, governance G
 
 	return &Oracle{
 		backend:          backend,
-		lastPrice:        config.Default,
```

Comment:
Would you consider if we pass in this default (or fallback? or initial?) gas tip from the command line?
- Currently it's set to UnitPrice in node/cn/backend.go. But the `oracle.lastPrice` is never used as pre-Magma suggestion in the current codebase. So config.Default doesn't have to be UnitPrice, rather it is the initial value for suggested 
- Geth sets `gpoParams.Default = config.Miner.GasPrice (from --miner.gasprice OR default 1 gwei)`
- Similarly, we can set `gpoParams.Default (from --gpo.initial-tip OR default 0 gwei)`

Also add a comment saying `lastPrice` is actually not last gasPrice, but last suggested tip. (Geth codebase still says `lastPrice` as well, so let's keep it this way)

Reply:
I agree with the idea but I think `lastPrice` set by `--gpo.initial-tip` will be almost useless since it'll be overridden to 0 by `isRelaxedNetwork` branch in most of case. So I think we can introduce command line config for later. How do you think?

Reply:
Sure, we can add it later.